kDrive/UI/Controller/Files/Search/SearchFiltersViewController.swift

```diff
@@ -64,6 +71,7 @@ class SearchFiltersViewController: UITableViewController {
         tableView.register(cellView: LocationTableViewCell.self)
         tableView.register(cellView: ManageCategoriesTableViewCell.self)
         tableView.register(cellView: SelectTableViewCell.self)
+        tableView.register(FileExtensionTextInputTableViewCell.self, forCellReuseIdentifier: "FileExtensionTextInputTableViewCell")
```

Comment:
I think we should use `tableView.register(cellView:)`

Reply:
I had issues with it during the rewrite in pure code of the cell. It does not work if there is no XIB.
I did it again just to be sure, and I had this exception : `"Could not load NIB in bundle: `

Reply:
Oh right, the method create a UINib... Thank you for trying again, I resolve the conversation.